src/HW1.py

```diff
@@ -0,0 +1,128 @@
+# Output
+import itertools
+
+print('Hello, world!')
+
+# Input, assignment
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+
+# For loop, built-in enumerate function, new style formatting
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print("iteration {iteration} is {name}".format(iteration=i, name=name))
+
+# Fibonacci, tuple assignment
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = [babies, parents + babies]
+
+# Dictionaries, generator expressions
+prices = {'apple': 0.40, 'banana': 0.50}
+my_purchase = {
+    'apple': 1,
+    'banana': 6}
+grocery_bill = sum(prices[fruit] * my_purchase[fruit]
+                   for fruit in my_purchase)
+print('I owe the grocer $%.2f' % grocery_bill)
+
+# Triple-quoted strings, while loop
+REFRAIN = '''
+%d bottles of beer on the wall,
+%d bottles of beer,
+take one down, pass it around,
+%d bottles of beer on the wall!
+'''
+bottles_of_beer = 9
+while bottles_of_beer > 1:
+    print(REFRAIN % (bottles_of_beer, bottles_of_beer,
+                     bottles_of_beer - 1))
+    bottles_of_beer -= 1
+
+
+# Classes
```

Comment:
Tasks should be separated with either one or two blank lines everywhere (it's not accurate to separate it with one line in one task and with two in another)